recipe/meta.yaml

```
@@ -35,21 +36,21 @@ requirements:
     - cuda-version {{ cuda_compiler_version }}
   run:
     - {{ pin_compatible('custatevec', max_pin='x') }}
-    - pennylane >=0.32
+    - pennylane >=0.33
     - python
     - {{ pin_compatible("cuda-version", min_pin="x", max_pin="x") }}
 test:
   imports:
-    - pennylane_lightning_gpu
+    - pennylane_lightning
   commands:
     - python -c "import pennylane as qml; qml.about()"
     - pip check
   requires:
-    - setuptools       # requirement from cvxpy
     - flaky
     - pip
     - pytest
     - pytest-mock
+    - setuptools
```

Is the comment still relevant or is this now a dependency of this package?

```suggestion
    - setuptools       # requirement from cvxpy
```

---

Good question, this was originally not a requirement "in principle", but the `pip check` would fail without it; if I remember correctly. Let's try to remove it.

---

I think I can further clean the test deps removing flaky, pytest, etc. since we don't run GPU tests.